### DISCUSSION TITLE
Mark posts as "bad tagged"

### DISCUSSION THREAD

**Discussion by MiguelMJ** [forem/forem#17164] [Feature Requests]
Hello, team!
I have a moderation related suggestion. If a user (maybe trusted users) could mark a post as "bad tagged", the moderators of the tags appearing in the post could more easily discover posts that don't belong in that category. This could reduce the number of posts using tags only for attention and not for the tag itself, which is quite a common thing in you know what tags....
Thanks to @michael-tharrington for suggesting me opening this ;)

**Comment by michael-tharrington** [comment]
Rock on! I really like this idea Miguel. Thanks for sharing it here! 

I could imagine these flagged posts somehow appearing in a tag mod's queue on https://dev.to/mod.

**Reply by salikadave** [reply]
Hey! @michael-tharrington

I believe [https://dev.to/mod](https://dev.to/mod) is only available to tag moderators. I want to explore the feasibility of finetuning a few models to automate some parts of the tag moderation (like spam or abuse or irrelevant content via text/images in a blog post, etc.). I would benefit if I had more data about bad tagged posts. Is it possible to make this link publicly accessible?

P.S. I have just applied to be a tag moderator. 😄
